lib/phoenix/channel/server.ex

```diff
@@ -193,7 +178,12 @@ defmodule Phoenix.Channel.Server do
   end
 
   defp join(socket, reply, parent, ref) do
-    PubSub.subscribe(socket.pubsub_server, self(), socket.topic, link: true)
+    PubSub.subscribe(socket.pubsub_server, self(), socket.topic,
+      link: true,
+      fastlane: {socket.transport_pid,
+                 socket.serializer,
+                 socket.channel.__phoenix_intercepts__()})
```

Comment:
Nitpick: Let's drop the `phoenix` from `__intercepts__` because we typically do prefix our metadata in Phoenix with `phoenix`. The only exception is the template reloading but only because it can be added to all modules.